lib/dark_worlds_server/bot/bot_client_supervisor.ex

```diff
@@ -0,0 +1,21 @@
+defmodule DarkWorldsServer.Bot.BotClientSupervisor do
+  use DynamicSupervisor
+  alias DarkWorldsServer.Bot.BotClient
+
+  def start_link(args) do
+    DynamicSupervisor.start_link(__MODULE__, args, name: __MODULE__)
+  end
+
+  def start_bot_clients(game_id, bot_count, config) do
+    Enum.each(1..bot_count//1, fn _ ->
+      {:ok, _child_pid} = DynamicSupervisor.start_child(__MODULE__, {BotClient, %{game_id: game_id, config: config}})
+    end)
+
+    :ok
+  end
```

Comment:
some things about this:

- bots would now be counted as regular players (in newrelic metrics). Are we ok with that? I think it would be nice to be able to differentiate them
- we're never checking the capacity on the game (neither here nor in the runner).

Reply:
> bots would now be counted as regular players (in newrelic metrics). Are we ok with that? I think it would be nice to be able to differentiate them

For good or bad this has been the intent from all this, and from a game perspective they are players. We will have a bot metric to have an idea, but maybe as a follow up we can add a param in the request to differentiate for telemetry purposes

Reply:
> we're never checking the capacity on the game (neither here nor in the runner).

In the good actors only case, `matching_coordinator` removed the need for that, it handles the capacity. 

In the bad actors case, in the future we should have a "secret" that only matched users know and thus will be the only ones allowed to connect to a game